scss/_default.assets-paths.scss

```diff
@@ -0,0 +1,3 @@
+$icons-path: "assets/icons/";
+$images-path: "assets/images/";
+$fonts-path: "assets/fonts/";
```

Comment:
it might be beneficial to make these themeable by adding a !default at the end (before the semicolon).

Reply:
fixed